src/webviews/extension-side/webviews/webview.ts

```diff
@@ -77,33 +78,24 @@ export abstract class Webview implements IWebview {
 
         const uriBase = this.webviewHost?.webview.asWebviewUri(this.options.cwd).toString();
         const uris = this.options.scripts.map((script) => this.webviewHost!.webview!.asWebviewUri(script));
-        const testFiles = await this.fs.getFiles(this.options.rootPath);
-
-        // This method must be called so VSC is aware of files that can be pulled.
-        // Allow js and js.map files to be loaded by webpack in the webview.
-        testFiles
-            .filter((f) => f.fsPath.toLowerCase().endsWith('.js') || f.fsPath.toLowerCase().endsWith('.js.map'))
-            .forEach((f) => this.webviewHost?.webview!.asWebviewUri(f));
```

Comment:
`testFiles` fails to be read in the browser. `vscfs.readDirectory` throws saying that `this.options.rootPath` (the `webview-side/viewers` directory) isn't a directory.

It could be something with the path, but this doesn't seem needed.
(I've asked @rchiodo , but please let me know if I misinterpreted anything)